feat(content): Culture Conversations for the Delvan Shipworm (#11109)
endless-sky/endless-sky: data/human/culture conversations.txt

			`	The other, a younger worker practically bouncing in his boots, grins behind his respirator. "Yeah! Little metal-chewing pests. They're native to Delve, infest the mines, the shipyards - anywhere there's metal to sink their teeth into. If we didn't spray every ship before takeoff, they'd be all over the galaxy by now." The veteran sighs. "They already are all over the galaxy. Because people aren't careful enough."`
			`	The rookie ignores him. "They start as tiny worms, barely bigger than your pinky, but once they find metal, they melt into it, make a cocoon that looks just like part of the ship! Bombacle, we call 'em. Then BAM! A few months later, the casing bursts open like a bomb! Bombacle!"`
			`	The veteran shakes his head. "What he means is, they're an invasive hazard if they get off-world. That's why we spray. No Shipworms, no problems."`
			`	The rookie gives his sprayer an enthusiastic shake. "And I get to torch every last one I find. You ever see one pop under a high-heat blast? Satisfying as hell." The veteran just mutters, "You'd think the novelty would wear off."`

> 			`	With another look at the freshly sprayed Bulk Freighter, you make a mental note to give your own ship a once-over before takeoff.`

Reviewer: ```suggestion
			action
				set "exotic animals: delvan shipworm known"
			`	With another look at the freshly sprayed Bulk Freighter, you make a mental note to give your own ship a once-over before takeoff.`
```

				goto ending
			label known
			`	The veteran lets out a long-suffering sigh. "Great. At least this one actually knows what they are." The rookie, on the other hand, practically lights up. "They're like space barnacles - until they hatch into wormy bombs! I was just telling him how fun it is to fry 'em!"`
			`	The veteran gestures around at the cavernous spaceport. "We don't get outbreaks here. This is Delve. They're Delvan Shipworms. They're native. Always have been. They infest the mines, the shipyards, anywhere with exposed metal. That's why every ship gets fumigated before takeoff - to keep 'em from spreading even more than they already have."`
			`	The rookie eagerly jumps back in. "You ever see one break out of a cocoon in real time? Casing splits and it blasts out like a grenade - it's wild!" The veteran just rubs his face. "Why do I even bother?"`
			`	With nothing more to add, you make a mental note to give your ship a once-over before takeoff as the workers continue their routine.`
			label ending
			`	You give the fumigation teams a final glance and make your way back to your ship, mindful of what might be lurking in the metal unseen.`
			action

> 				set "exotic animals: delvan shipworm known"

Reviewer: Ah, one question. Shouldn't this be moved to where it won't be set after `label known`?

> 			action
> 				set "exotic animals: delvan shipworm known"

Reviewer: ```suggestion
```

> 			`	The rookie gives his sprayer an enthusiastic shake. "And I get to torch every last one I find. You ever see one pop under a high-heat blast? Satisfying as hell." The veteran just mutters, "You'd think the novelty would wear off."`
> 			`	With another look at the freshly sprayed Bulk Freighter, you make a mental note to give your own ship a once-over before takeoff.`
> 				goto ending
> 			label known
> 			`	The veteran lets out a long-suffering sigh. "Great. At least this one actually knows what they are." The rookie, on the other hand, practically lights up. "They're like space barnacles - until they hatch into wormy bombs! I was just telling him how fun it is to fry 'em!"`
> 			`	The veteran gestures around at the cavernous spaceport. "We don't get outbreaks here. This is Delve. They're Delvan Shipworms. They're native. Always have been. They infest the mines, the shipyards, anywhere with exposed metal. That's why every ship gets fumigated before takeoff - to keep 'em from spreading even more than they already have."`
> 			`	The rookie eagerly jumps back in. "You ever see one break out of a cocoon in real time? Casing splits and it blasts out like a grenade - it's wild!" The veteran just rubs his face. "Why do I even bother?"`
> 			`	With nothing more to add, you make a mental note to give your ship a once-over before takeoff as the workers continue their routine.`
> 			label ending
> 			`	You give the fumigation teams a final glance and make your way back to your ship, mindful of what might be lurking in the metal unseen.`
> 			action
> 				set "exotic animals: delvan shipworm known"

Author: ```suggestion
			`	The rookie gives his sprayer an enthusiastic shake. "And I get to torch every last one I find. You ever see one pop under a high-heat blast? Satisfying as hell." The veteran just mutters, "You'd think the novelty would wear off."`
			action
				set "exotic animals: delvan shipworm known"
			`	With another look at the freshly sprayed Bulk Freighter, you make a mental note to give your own ship a once-over before takeoff.`
				goto ending
			label known
			`	The veteran lets out a long-suffering sigh. "Great. At least this one actually knows what they are." The rookie, on the other hand, practically lights up. "They're like space barnacles - until they hatch into wormy bombs! I was just telling him how fun it is to fry 'em!"`
			`	The veteran gestures around at the cavernous spaceport. "We don't get outbreaks here. This is Delve. They're Delvan Shipworms. They're native. Always have been. They infest the mines, the shipyards, anywhere with exposed metal. That's why every ship gets fumigated before takeoff - to keep 'em from spreading even more than they already have."`
			`	The rookie eagerly jumps back in. "You ever see one break out of a cocoon in real time? Casing splits and it blasts out like a grenade - it's wild!" The veteran just rubs his face. "Why do I even bother?"`
			`	With nothing more to add, you make a mental note to give your ship a once-over before takeoff as the workers continue their routine.`
			label ending
			`	You give the fumigation teams a final glance and make your way back to your ship, mindful of what might be lurking in the metal unseen.`
```

			label "sestor beaten at farpoint"
			`	Web comments and news anchors seem to be particularly excited to comment on the recent victory of the Navy over the mysterious armada of robotic warships that invaded the Far North. Though at a great cost to the northern fleet garrisons, the invading fleets were successfully stopped at the Farpoint base, preventing the ships from causing any further harm. Veterans of the Farpoint battle seem to be hoarding the spotlight, giving emotional reports of their memory of the fight, and of the sacrifices they went through to achieve victory.`
				goto end

mission "Exotic Life: Delve"
	minor
	source Delve
	to offer

> 		random < 20

Author: My thought was that the workers would spray most ships so that it would make sense for a higher chance of you seeing them -- and you would still need to visit the spaceport on Delve 10 times for this to have a ~90% chance of showing up. Happy to tweak it down to the normal 10 though if you think that's better.

> 		random < 20

Reviewer 2: Makes sense.

	on offer
		conversation

> 			`The air in Delve's spaceport is thick with the scent of fuel, damp stone, and something acrid - chemical, artificial. As you walk toward your ship, you notice teams of workers dressed in biohazard gear moving between the departing vessels in pairs, bulky sprayers strapped to their backs, nozzles hissing as they coat hulls in a fine mist. Each ship gets a thorough pass before takeoff, the workers checking crevices and landing gear with practiced efficiency.`

Reviewer 3: Hmm... the `chemical, artificial` here seems slightly awkward, although that might just be me. It works pretty well with your writing style, though.

Author: I agree after rereading this. Does something like the below flow better if I take out the dash and spell out the acrid smell?

```suggestion
			`The air in Delve's spaceport is thick with the scent of fuel, damp stone, and an acrid tang from industrial cleaners lingering in the recirculated ventilation. As you walk toward your ship, you notice teams of workers dressed in biohazard gear moving between the departing vessels in pairs, bulky sprayers strapped to their backs, nozzles hissing as they coat hulls in a fine mist. Each ship gets a thorough pass before takeoff, the workers checking crevices and landing gear with practiced efficiency.`
```

> 			`The air in Delve's spaceport is thick with the scent of fuel, damp stone, and an acrid tang from industrial cleaners lingering in the recirculated ventilation. As you walk toward your ship, you notice teams of workers dressed in biohazard gear moving between the departing vessels in pairs, bulky sprayers strapped to their backs, nozzles hissing as they coat hulls in a fine mist. Each ship gets a thorough pass before takeoff, the workers checking crevices and landing gear with practiced efficiency.`

Reviewer 4: While the air in the mining cities certainly seems like it will have been through a lot of machinery, the spaceport is at the surface, according to the description:
https://github.com/endless-sky/endless-sky/blob/191a37165619707ad7227431b8a84c5ee6570d40/data/map%20planets.txt#L1217
Since it's described as a "hollow", and not a "cave", and is "carved out of the side of the mountain", not carved out of the interior, I expect the opening is quite enormous (it even says as much). I'm imagining a chunk taken out of hte side of the mountain, not a tunnel leading into a large empty space within. It doesn't strike me as somewhere that would rely on recirculated air. Maybe we need artificial ventilation for hangar caves, but I don't imagine even that would need to be recirculated air.

(Although, I'm not sure the air in the mining cities is all that recirculated, either. The description says:
https://github.com/endless-sky/endless-sky/blob/191a37165619707ad7227431b8a84c5ee6570d40/data/map%20planets.txt#L1216
The key point here being "mechanical ventilation from the surface." If it's from the surface, it's not being recirculated.)

Author: Thanks for taking a look at these! I definitely read the spaceport as being a hollowed out underground city, and agree that your reading is probably more correct. My thought on the recirculation was that the spaceport's air would be scrubbed for ship fumes even if not done elsewhere, but as you mentioned the planet description explicitly calls out that the air is all from the surface even kilometers deep. Does this work better -- the scent is now predominately crisp mountain air with the fuel/industrial cleaners (now in a general sense) being occasional whiffs carried by wind?

```suggestion
			`The air in Delve's spaceport is crisp, carrying the mineral tang of weathered stone alongside occasional whiffs of fuel and lingering industrial cleaners. As you walk toward your ship, you notice teams of workers dressed in biohazard gear moving between the departing vessels in pairs, bulky sprayers strapped to their backs, nozzles hissing as they coat hulls in a fine mist. Each ship gets a thorough pass before takeoff, the workers checking crevices and landing gear with practiced efficiency.`
```

Reviewer 4: Yes, I think this works better.

			choice
				`	(Ask them what they're doing.)`
				`	(Ignore them and continue on your way.)`
					decline
				`	"Shipworm outbreak?"`
					to display
						has "exotic animals: delvan shipworm known"
					goto known

> 			`	You approach a pair of workers as they finish spraying down a Bulk Freighter. One of them, a grizzled veteran with deep lines around his eyes, barely glances at you. "Shipworms," he mutters, already moving to the next ship.`

Reviewer 3: If he's barely glancing at you, why does he bother to talk? I think you're missing a sentence or clause here explaining how you asked them what was going on, so it reads like you ask them *then* walk over, which is backwards. Maybe something like:
```suggestion
			`	You approach a pair of workers as they finish spraying down a Bulk Freighter and ask what they're doing. One of them, a grizzled veteran with deep lines around his eyes, barely glances at you. "Shipworms," he mutters, already moving to the next ship.`
```

			`	The other, a younger worker practically bouncing in his boots, grins behind his respirator. "Yeah! Little metal-chewing pests. They're native to Delve, infest the mines, the shipyards - anywhere there's metal to sink their teeth into. If we didn't spray every ship before takeoff, they'd be all over the galaxy by now." The veteran sighs. "They already are all over the galaxy. Because people aren't careful enough."`
			`	The rookie ignores him. "They start as tiny worms, barely bigger than your pinky, but once they find metal, they melt into it, make a cocoon that looks just like part of the ship! Bombacle, we call 'em. Then BAM! A few months later, the casing bursts open like a bomb! Bombacle!"`

> 			`	The veteran shakes his head. "What he means is, they're an invasive hazard if they get off-world. That's why we spray. No Shipworms, no problems."`

Reviewer 4: Does "Shipworm" need to be capitalised? We don't capitalise "worm", or the names of other animals.

Author: Good point -- uncapitalized "shipworm." Left "Delvan" capitalized as that's a location and follows the naming scheme of other [animals named after locations](https://commons.wikimedia.org/wiki/Category:Animals_named_after_places) that I looked at.

			`	The rookie gives his sprayer an enthusiastic shake. "And I get to torch every last one I find. You ever see one pop under a high-heat blast? Satisfying as hell." The veteran just mutters, "You'd think the novelty would wear off."`
			action
				set "exotic animals: delvan shipworm known"
			`	With another look at the freshly sprayed Bulk Freighter, you make a mental note to give your own ship a once-over before takeoff.`
				goto ending
			label known

> 			`	The veteran lets out a long-suffering sigh. "Great. At least this one actually knows what they are." The rookie, on the other hand, practically lights up. "They're like space barnacles - until they hatch into wormy bombs! I was just telling him how fun it is to fry 'em!"`

Reviewer 4: If we choose the option that leads here, these two guys haven't been described to us.
The reader will have no context for "the veteran".

Author: How's this?
```suggestion
			label known
			`	One of them, a seasoned veteran with weary eyes, lets out a long-suffering sigh. "Great. At least this one actually knows what they are." In contrast, the other - a fresh-faced worker brimming with energy - practically lights up. "They're like space barnacles - until they hatch into wormy bombs! I was just telling him how fun it is to fry 'em!"`
```

Reviewer 4: Yes, I think this resolves it.

			`	The veteran gestures around at the cavernous spaceport. "We don't get outbreaks here. This is Delve. They're Delvan Shipworms. They're native. Always have been. They infest the mines, the shipyards, anywhere with exposed metal. That's why every ship gets fumigated before takeoff - to keep 'em from spreading even more than they already have."`
			`	The rookie eagerly jumps back in. "You ever see one break out of a cocoon in real time? Casing splits and it blasts out like a grenade - it's wild!" The veteran just rubs his face. "Why do I even bother?"`
			`	With nothing more to add, you make a mental note to give your ship a once-over before takeoff as the workers continue their routine.`
			label ending

> 			`	You give the fumigation teams a final glance and make your way back to your ship, mindful of what might be lurking in the metal unseen.`

Reviewer 3: The mental note is probably good, but telling me I'm mindful of what might be lurking is a little close to telling me what I'm thinking. Maybe if the rookie was more enthusiastic in describing the worms in horrific detail?

				decline

mission "Exotic Life: Ada"
	minor
	source Ada
	to offer
		random < 10
	on offer
		conversation

> 			`You make your way through a dimly lit section of the spaceport, where the usual hum of activity is dampened by an eerie stillness. Large, sealed tarps stretch across the area, some draped over cargo crates, others covering entire sections of the corridor. One enormous tarp seems to be covering a whole ship. Each one bears the same stark warning in bold lettering: 'DELVAN SHIPWORM FUMIGATION.' The air has a faint chemical tang to it, and ahead, two figures move methodically between the tarps.`

Reviewer 3: American English
```suggestion
			`You make your way through a dimly lit section of the spaceport, where the usual hum of activity is dampened by an eerie stillness. Large, sealed tarps stretch across the area, some draped over cargo crates, others covering entire sections of the corridor. One enormous tarp seems to be covering a whole ship. Each one bears the same stark warning in bold lettering: "DELVAN SHIPWORM FUMIGATION." The air has a faint chemical tang to it, and ahead, two figures move methodically between the tarps.`
```

> 			`You make your way through a dimly lit section of the spaceport, where the usual hum of activity is dampened by an eerie stillness. Large, sealed tarps stretch across the area, some draped over cargo crates, others covering entire sections of the corridor. One enormous tarp seems to be covering a whole ship. Each one bears the same stark warning in bold lettering: "DELVAN SHIPWORM FUMIGATION." The air has a faint chemical tang to it, and ahead, two figures move methodically between the tarps.`

Reviewer 4: You call this a corridor, but also, there's a ship in here? Seems more like a hangar than a corridor.
It also seems kind of strange that I would even have access to this area? I feel like it would make more sense if we were looking in through some sort of transparent barrier across a large opening, perhaps a cargo door at the back of the hangar?

Author: Yeah, that is a bit incongruous. Using your hangar and cargo door idea, how does this look? I wanted the door to be cracked open and the workers walking towards you for the questioning to not feel like you're breaking in to chat, and made the tarp covering a ship more vague to be merely large enough that it _could_ be big enough to cover a ship.

```suggestion
			`You make your way through a dimly lit section of the spaceport to a near-empty hangar, where the usual hum of activity is dampened by an eerie stillness. The back of the hangar has a reinforced glass cargo door cracked open, perhaps by mistake. You peer inside to see large, sealed tarps stretch across the area, some draped over cargo crates, others covering entire sections. One enormous tarp appears big enough to be covering a whole ship. Each one bears the same stark warning in bold lettering: "DELVAN SHIPWORM FUMIGATION." The air has a faint chemical tang to it, and ahead, two figures move methodically between the tarps towards the door.`
```

Reviewer 4: Yes, I think this works better.

			`	They are covered head to toe in protective suits, thick gloves, and respirators obscuring their faces. One carries a handheld sprayer, releasing a fine mist into the air, while the other checks a monitor strapped to their arm. Their boots crunch softly against the floor as they pass.`
			choice
				`	(Ask them what's going on.)`
				`	(Ignore them and continue on your way.)`
					decline
				`	"Nasty little wrigglers, aren't they?"`
					to display
						has "exotic animals: delvan shipworm known"
					goto known
			`	Curiosity gets the better of you, and you ask what this is all about, gesturing at the tarps. One of them pauses, their voice muffled by the respirator. "Delvan Shipworm outbreak. Nasty infestation. We're exterminating them before they spread to other ships or the city." You can see small scorch marks on the floor nearby - signs of something being burned away.`
			choice
				`	"What's a Delvan Shipworm?"`
				`	"Nasty bombacles, aren't they?"`
					to display
						has "exotic animals: delvan shipworm known"
					goto known

> 				`	"Well, good luck with that. I must be on my way."`

Reviewer 3: "I must be on my way" sounds awfully formal. Is there a slightly more casual way to say it?

Author: As a matter of fact I think there is:
```suggestion
				`	"Well, good luck with that. I should get going."`
```

					decline
			`	The worker with the monitor barely hesitates before responding. "Delvan Shipworms are an invasive species from Delve. They start as larvae, tiny worms-"`
			`	"Bombacles!" the one with the sprayer interjects, voice distorted through the respirator. "They latch on ships, grow into barnacles, then they're bombs!"`
			`	The first worker exhales, shaking their head. "They're more than that. The larvae attach to metal - usually ship hulls, but it can be mining equipment or anything really, and secrete an enzyme that corrodes the surface, breaking it down so they can form a cocoon around themselves. This builds up a high-pressure gas, so the outer shell hardens to compensate, blending in with the surrounding metal."`
			`	"Yeah, until - BOOM! Three months later, it blasts open like a grenade and out come a dozen disgusting worms!"`
			`	"You make it sound like a horror movie."`

> 			`	"It is a horror movie if you're next to one when it hatches." The sprayer worker turns back to you. "Writhing wrigglers that leave a hole the size of your palm in whatever they break out of. Plenty of ships have lost pressure 'cause of them."`

Reviewer 3: ```suggestion
			`	"It is a horror movie if you're next to one when it hatches." The sprayer-wielding worker turns back to you. "Writhing wrigglers that leave a hole the size of your palm in whatever they break out of. Plenty of ships have lost pressure 'cause of them."`
```

Sprayer worker is a bit confusing and sounds repetitive even though it's not.

			`	The first worker sighs. "That's why we're dealing with it now. We don't let them spread, we don't get people vented into space."`
			action
				set "exotic animals: delvan shipworm known"
			`	Your gaze drifts back to the tarps, imagining what might be skittering beneath them. You nod in thanks and step away, making a mental note to double-check your ship's hull before takeoff.`
				goto ending
			label known
			`	The worker with the sprayer perks up. "Finally, someone who gets it!" they say, shaking their sprayer for emphasis.`

> 			`	The other groans. "They are not barnacles and they are not bombs. They are a highly adaptive invasive species with a complex-"`

Reviewer 4: If we came here from the first choice, no one has mentioned "barnacles" or "bombacles" in this conversation.

Author: Just changed "wrigglers" --> "bombacles" in the first choice
